portals/publisher/source/src/app/data/AuthManager.js

```diff
@@ -19,6 +19,7 @@
 import qs from 'qs';
 import CONSTS from 'AppData/Constants';
 import Configurations from 'Config';
+import _ from 'lodash';
```

Comment:
Shall we use the [lodash.intersectionby](https://www.npmjs.com/package/lodash.intersectionby) package similar to [what we have here](https://github.com/wso2/apim-apps/blob/main/portals/publisher/package.json#L53-L54)